asset/js/setup.js

```diff
@@ -19,6 +19,18 @@ function flattenModals() {
   });
 }
 
+/**
+ * Fixing the top navbar would break anchor navigation,
+ * by creating empty spans above the <h> tag we can prevent
+ * the headings from being covered by the navbar.
+ */
+function setupDummySpans() {
+  jQuery('<span class="anchor"></span>').insertBefore('h1, h2, h3, h4, h5, h6');
```

Comment:
There are some constants in `src/constants.js` that you can consider using for this